src/conditionWait.ts

```diff
@@ -0,0 +1,56 @@
+import { Locator } from "playwright";
+
+export const conditionValidations = {
+    PRESENT: 'present',
+    // CLICKABLE: 'clickable',
+    VISIBLE: 'visible',
+    INVISIBLE: 'invisible',
+    // ENABLED: 'enabled',
+    // DISABLED: 'disabled'
+}
+
+const notClause = '(not )?';
```

Comment:
maybe it will be a good idea to name that constant as constants, e.x. `NOT_CLAUSE` and etc?

Reply:
make sense, will do it with next update